boards/arm/nrf52840_pca10056/nrf52840_pca10056.dts

```diff
@@ -175,19 +175,29 @@
 		};
 		slot0_partition: partition@c000 {
 			label = "image-0";
-			reg = <0x0000C000 0x000069000>;
+			reg = <0x0000C000 0x000067000>;
 		};
-		slot1_partition: partition@75000 {
+		slot1_partition: partition@73000 {
 			label = "image-1";
-			reg = <0x00075000 0x000069000>;
+			reg = <0x00073000 0x000067000>;
 		};
-		scratch_partition: partition@de000 {
+		scratch_partition: partition@da000 {
 			label = "image-scratch";
-			reg = <0x000de000 0x0001e000>;
+			reg = <0x000da000 0x0001e000>;
 		};
-		storage_partition: partition@fc000 {
+
+		/*
+		 * The flash starting at 0x000f8000 and ending at
+		 * 0x000fffff is reserved for use by the application.
+		 */
+
+		/*
+		 * If enabled, partition for FCB/NFFS/NVS will be created in
```

Comment:
This is inaccurate, the partition is there regardless and not when only when enabled (partition can't be enabled/disabled).

Reply:
Thnaks for spot this.
maybe repharse to sth like: storage partition can be used by FCB/NFFS/NVS once one of those is enabled.